read_motif_v6.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please add your code to the `htsinfer/infer_read_layout.py` module rather than its own module